tests/jerry/fail/regression-test-issue-2058.js

```diff
@@ -12,9 +12,4 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-try {
-  /?:/
-  assert(false);
-} catch (e) {
-  assert(e instanceof SyntaxError);
-}
+/?:/
```

Comment:
Would it make sense to have these tests in an eval call and have correct error type check?

Reply:
Sure!